tests/Integration/run-test.sh

```diff
@@ -11,7 +11,27 @@ if [ ! -e "$TESTSCRIPT" ] ; then
     exit
 fi
 
-
 # sleep is necessary, otherwise the LDAP server cannot be connected to, yet.
-setup-scripts/start.sh && sleep 5 && php -f "$TESTSCRIPT"
-setup-scripts/stop.sh
+WEBUSER=`ls -ld ../../../../config/config.php | awk '{print $3}'`
+OUT=""
+setup-scripts/start.sh > /dev/null && sleep 5 && \
+    if [ "$WEBUSER" == "travis" ]; then
+	../../../../occ app:enable user_ldap && \
```

Comment:
I usually define a shell variable `OCC=../../../../occ` at the top of the shell script to avoid repeating the ugly dots.